exampleCourse/questions/element/template/question.html

```diff
@@ -0,0 +1,48 @@
+<p>
+  This page shows has example uses of the <code>pl-template</code>
+  element, including uses of recursive templates.
+</p>
```

Comment:
To me a recursive template would seem to denote a template that imports _itself_. I don't see that happening here, but I do see templates importing _other_ templates. Is that right, or am I missing something?

Either way, I don't think "recursive" is the best thing to call this in documentation. I think it would be much more clear to just say "...including uses of templates that import other templates."

Reply:
That's correct, templates don't call themselves. I'll change the language to say "nested" templates.

Reply:
I don't think that be obvious to all users what that is supposed to mean. Why can't you just say "templates that import/use other templates"?